frontend/src/components/ListCar.jsx

```diff
@@ -1,18 +1,23 @@
 import React, { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
 import "./style/ListCar.scss";
 import { Link } from "react-router-dom";
 import axios from "axios";
 const { apiSite } = require("../conf");
 
 function ListCar() {
   const [nextMaintenance, setnextMaintenance] = useState([]);
-
+  const token = useSelector(state => state.user.token);
   useEffect(() => {
-    axios.get(`${apiSite}/vehicule/1/nextmaintenance`).then(({ data }) => {
-      const lvls = calcLevels(data);
-      const filtered = filterFamilies(lvls);
-      setnextMaintenance(filtered);
-    });
+    axios
+      .get(`${apiSite}/vehicule/1/nextmaintenance`, {
```

Comment:
So is this one

Reply:
it's a bugfix branch, this kind of problem will be resolve in another branch